tests/unit/common/test_artifact_source.py

```diff
@@ -26,6 +29,11 @@ def _init(self) -> None:
         source_file_sample = current_file_path / "./artifacts.json"
         self._artifact_source.load(str(source_file_sample))
 
+    def test_loads_from_http(self):
+        artifact_source = ArtifactSources()
+        with pytest.raises(ManifestDownloadError):
+            artifact_source.load("http://holoscan")
```

Comment:
Does not matter http vs https? The next new test uses https though for mock_get

Reply:
It doesn't matter, but I've renamed and updated the test to use a mock.